app/views/pages/moving-guide/civilians.html.erb

```diff
@@ -1,11 +1,331 @@
+            <strong>Pro-Tip:</strong> Enlist your family or your Sponsor for help during this during this phase.
+          <% end %>
+        </div>
+      </div>
+
+      <p>Finally, you have a couple of options during the delivery phase about what items you want the delivery crew to unpack.</p>
+
+      <ul>
+	      <li>A full unpack: Movers empty the boxes of all their contents into the specified rooms.</li>
+	      <li>A partial unpack: Movers only open boxes in select areas.</li>
+        <li>Movers reassemble furniture that they took apart at the origin (beds, couches, etc.)</li>
+      </ul>
+    </div>
+
+    <div id="clean-up" class="article-section">
+      <h4>Clean Up, Claims &amp; Customer Survey</h4>
+
+      <p>
+        <strong><em>Clean-Up</em></strong><br>
+        Once you have emptied out all your household goods from the boxes, call the local moving company and set up a time for them to come back out and remove any old boxes, packing material, etc. They are responsible for removing it from the residence one time after delivery.
+      </p>
+
+      <%= render layout: 'shared/pro_tip' do %>
+        <strong>Pro-Tip:</strong> Save time by piling up all the boxes and moving supplies in one location!
+      <% end %>
+
+      <p>
+        <strong><em>Claims</em></strong><br>
+        While we hope all of your items arrived in one piece, the reality is that isn’t always the case. Sign in to <%= link_to 'Defense Personal Property System', 'https://eta.sddc.army.mil/ETASSOPortal/default.aspx' %> and complete this part of the process. When you log in to the system, you can expect to see two paths - Lost/Damage Report and Claims Submission.
+      </p>
+
+      <p>A Loss/Damage Report is your way to notify the moving company that some of your items are missing or were damaged during the delivery process. This must be done within 75 days of your delivery date. If the items cannot be found or they cannot fix the damage, you must file a claim to be reimbursed! Remember... a Loss/Damage Report is NOT a claim!</p>
+
+      <p>If you did not submit a Loss/Damage Report, you will need to file your claim within 75 days of delivery. If you submitted a Loss/Damage Report within the 75 day window, you will have up to 9 months to file a claim for the Full Replacement Value of the item you are claiming. If the claim is filed more than 9 months from the delivery date, you will only be eligible for depreciated value up to two years from delivery.</p>
+
+      <p>
+        <strong><em>Customer Satisfaction Survey</em></strong><br>
+        The survey asks some questions about your experience is read by actual humans! Be upfront and honest in your feedback, because if there is a negative trend on a moving company they can be suspended temporarily or indefinitely if they are providing poor service. On the flip side, if you find a moving company that did an excellent job, please put that in the survey as well to help out your colleagues!
```

Comment:
Either "The survey **that** asks some questions about your experience is read by actual humans!" or "The survey asks some questions about experience**, and** is read by actual humans!"

Reply:
"If there is a negative trend on a moving company _comma_"

Reply:
There's another _if_ clause at the end of that sentence. There should be only the if clause at the begin or at the end, not both.